.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,102 @@
+name: build_and_test
+
+# Only run tests for main branch or if the PR has relevant changes
+on: 
+  push:
+    branches:
+      - main
+  pull_request:
+
+concurrency:
+  group: build_and_test-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+env:
+  RESULT_PATH: testresults
+  PYTHON_VERSION: 3.8.10
+  REQUIREMENTS_PATH: "tests/requirements.txt"
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the codebase.
+        uses: actions/checkout@v2
+
+      - name: Setup bundle
+        env:
+          PULL_CACHE: yes
+          AGENT_VERSION: latest
+        run: |
+          make bundle
+          mkdir -p /tmp/workspace
```

Comment:
Let's rename `/tmp/workspace` to `./dist`, and update the paths below.  We only used `/tmp/workspace` in circleci because we didn't want to cache the entire project's workspace.

Reply:
Make sense. Changes made.